libsplinter/src/service/network/interconnect/mod.rs

```diff
@@ -13,3 +13,707 @@
 // limitations under the License.
 
 mod error;
+mod scml;
```

Comment:
This module name is a bit cryptic; what does it mean?

Reply:
renamed